lib/gemstash/migrations/05_authorization_names.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table :authorizations do
+      add_column :name, String, :size => 191
```

Comment:
Size seems oddly specific, am I missing something?

Reply:
Ha! We share that thought. I was honestly just duplicating the [existing pattern for String columns](https://github.com/rubygems/gemstash/blob/main/lib/gemstash/migrations/02_authorizations.rb#L7-L8) in the authorizations table, because they're magic numbers there as well. This number means nothing to me. Would you prefer something different?

Reply:
I learned, by inspecting the git blame navigation: https://github.com/rubygems/gemstash/pull/64